content/articles/tidyverse_1.2.0.Rmd

```diff
@@ -0,0 +1,75 @@
+---
+title: tidyverse 1.2.0
+slug: tidyverse-1.2.0
+description: >
+  A new version of the tidyverse package is now on CRAN.
+author: Mara Averick
+date: 2017-11-15
+photo:
+  url: https://pixabay.com/en/universe-sky-star-space-all-1566159/
+  author: Gerd Altmann
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("tidyverse")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+The release of [tidyverse 1.2.0](https://github.com/tidyverse/tidyverse/releases) 
+on CRAN (now 1.2.1, following some minor fixes) includes updates in tidyverse 
+package membership, as well as to tidyverse functions themselves.
+
+The latest version of tidyverse can be installed with:
+
+```{r tidyverse_install, eval = FALSE}
+install.packages("tidyverse")
+```
+
+## Changes in tidyverse membership
+
+### Addition of the reprex package
+
+Though it's been a mainstay tidyverse [Help](https://www.tidyverse.org/help/), 
```

Comment:
"Help" -> "of help"?  Not quite sure what you're trying to say here

Reply:
It's clearly a cry for help!